src/sqlfluff/core/templaters/jinja.py

```diff
@@ -513,6 +513,7 @@ class DummyUndefined(jinja2.Undefined):
     """
 
     def __init__(self, name):
+        super().__init__()
```

Comment:
I forgot to call the base class constructor, resulting in undefined base class fields, causing 💥.